week-2/L-array-methods-2/exercise3.js

```diff
@@ -4,23 +4,22 @@
   Complete the function below to check if a country is in the UK
 */
 
-var ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
 
 function isInUK(country) {
-  return; // complete this statement
+  var ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
+  return ukNations.includes(country); // complete this statement
```

Comment:
Great! Any reason you put the variable inside the function? It would have worked  anyways :)